docs/conf.py

```diff
@@ -0,0 +1,39 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = 'CppInterOp'
+copyright = '2023, Vassil Vassilev'
+author = 'Vassil Vassilev'
+release = 'Dev'
+
+# -- General configuration ---------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration
+
+extensions = []
+
+templates_path = ['_templates']
+exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
+
+
+
+# -- Options for HTML output -------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
+
+html_theme = 'alabaster'
+html_static_path = ['_static']
+
+INTEROP_ROOT = '..'
+
+# html_extra_path = [INTEROP_ROOT + '/build/docs/doxygen/html']
```

Comment:
Why is this commented out?
And also, `html_extra_path` directory probably should not contain `index.html` file. Otherwise,
readthedocs will get confused which page should be used when visiting the website root.

Reply:
In my next patch to fix the issue of doxygen, I have added the ```html_extra_path``` https://github.com/compiler-research/CppInterOp/compare/main...Krishna-13-cyber:CppInterOp:CppInterOp-Doxygen-Implementation#diff-85933aa74a2d66c3e4dcdf7a9ad8397f5a7971080d34ef1108296a7c6b69e7e3R54 but still the issue persists .
The index.html for sphinx is being used as website root as we can see the output. The build directory of doxygen which consists of index.html is implicitly generated by default, so are you suggesting to modify the config to remove the index generation?

Current Status including the  ```html_extra_path```:
https://readthedocs.org/projects/cppinterop-docs/builds/21074694/

Reply:
I am suggesting to make the `html_extra_path` to something like, `INTEROP_ROOT + '/build/docs'`. This way we can
access internal documentation at `${WebsiteRoot}/doxygen/html`.